core/src/main/scala/org/bitcoins/core/gcs/GolombFilter.scala

```diff
@@ -84,6 +84,16 @@ case class GolombFilter(
 
     matchesHash(hash)
   }
+
+  def matchesAnyHash(hashes: Vector[UInt64]): Boolean =
```

Comment:
Could you leave a TODO and open an issue to refactor this method to do as is specified here https://github.com/bitcoin/bips/blob/master/bip-0158.mediawiki#golomb-coded-set-multi-match

Probably link to https://github.com/bitcoin-s/bitcoin-s/issues/523 in that new issue since they are related